api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessor.java

```diff
@@ -98,6 +99,8 @@ private void handleDecision(Claim claim, PaymentCycle previousPaymentCycle, Paym
             } else {
                 handleNoLongerEligibleForSchemeAsNoChildrenAndNotPregnant(claim, decision.getDateOfBirthOfChildren());
             }
+
+            setCardStatusToPendingCancellation(claim);
```

Comment:
I don't think this is in the right place. This seems to me to always set active claims to pending_expiry?

Reply:
I take it back. This is in the right place.